OpenRA.Mods.Common/Traits/ExternalResourceMultiplier.cs

```diff
@@ -0,0 +1,76 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Mods.Common.Effects;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Gives additional cash when resources are delivered to refineries.")]
+	public class ExternalResourceMultiplierInfo : ConditionalTraitInfo
+	{
+		[FieldLoader.Require]
+		[Desc("Percentage value of the resource to grant as cash.")]
+		public readonly int Modifier;
+
+		public readonly bool ShowTicks = true;
+		public readonly int TickLifetime = 30;
+		public readonly int TickVelocity = 2;
+		public readonly int TickRate = 10;
```

Comment:
It is not really obvious to me (from just looking at the Info) what this does. What about `Interval` with a description saying it is the interval for the floating text?

Reply:
Name is coming from Refinery trait. If i change this imo, i should change the other too.